src/index.ts

```diff
@@ -528,7 +561,10 @@ const main = async ({
     await buildVercel();
   }
 
-  await transform({ experimentalMinify });
+  await Promise.all([
+    transform({ experimentalMinify }),
+    generateMetadataFromConfig(),
```

Comment:
Cloud you move the `generateMetadataFromConfig` function (alongside the other function) closer to here? (for example above `main`)

I think it would make things clearer to define the function(/s) closer to where it is used.

(PS: we need to split this file into multiple files at some point but that's a slightly lesser priority so for now at least if you could move the functions that's be great 🙏 )

Reply:
I wouldn't `await Promise.all()` these. There's no serious performance benefit, they're not conceptually similar, and it would result in intermingled messages if either of these functions log something. Better to just `await` each, one-after-the-other.

Reply:
Both done as well.